src/lib.rs

```diff
@@ -343,6 +346,11 @@ impl Config {
         self.tls_ctx.set_verify(verify);
     }
 
+    /// Configures whether to send GREASE values.
+    pub fn send_grease(&mut self, grease: bool) {
```

Comment:
Should call this `enable_grease()`, or just `grease()`.

Reply:
done, new name: `grease()`